images/Makefile

```diff
@@ -39,6 +48,12 @@ build-rootfs:
 		else \
 			echo "Building rootfs image: $$image"; \
 			docker build -t $$image $$rootfs_dir; \
+			if [ "$(PUSH)" = "true" ]; then \
+				echo "Pushing image: $$image"; \
+				docker push $$image; \
+			fi; \
+			docker image rm -f $$image; \
```

Comment:
Since images are big, we don't want to fill up the runner node disk space; therefore after each build and eventual push, clean up images.